libs/constrained-generators/src/Constrained/Base.hs

```diff
@@ -1536,7 +1544,7 @@ simplifyPred = \case
     t' -> GenHint h t'
   Subst x t p -> simplifyPred $ substitutePred x t p
   Assert es t -> assertExplain es (simplifyTerm t)
-  Reify tm f b -> mkReify (simplifyTerm tm) f (simplifyBinder b)
+  Reifies t' t f -> Reifies (simplifyTerm t') (simplifyTerm t) f
```

Comment:
This is where we hope (simplifyTerm t)  will eventually be come a literal?

Reply:
Yes, that's right.